docs/en/peripherals/esc_motors.md

```diff
@@ -3,80 +3,42 @@
-
-PWM ESCs communicate using a periodic pulse, where the _width_ of the pulse indicates the desired power level.
-The pulse wdith typically ranges between 1000uS for zero power and 2000uS for full power.
-The periodic frame rate of the signal depends on the capability of the ESC, and commonly ranges between 50Hz and 490 Hz (the theoretical maximum being 500Hz for a very small "off" cycle).
-A higher rate is better for ESCs, in particular where a rapid response to setpoint changes is needed.
-For PWM servos 50Hz is usually sufficient, and many don't support higher rates.
-
-![duty cycle for PWM](../../assets/peripherals/esc_pwm_duty_cycle.png)
-
-In addition to being a relatively slow protocol PWM ESCs require [calibration](../advanced_config/esc_calibration.md) because the range values representing low and high values can vary significantly.
-Unlike [dshot](#dshot) and [DroneCAN ESC](#dronecan) they do not have the ability to provide telemetry and feedback on ESC (or servo) state.
-
-Setup:
-
-- [ESC Wiring](../peripherals/pwm_escs_and_servo.md)
-- [PX4 Configuration](../peripherals/pwm_escs_and_servo.md#px4-configuration)
-- [ESC Calibration](../advanced_config/esc_calibration.md)
+| ESC Device                   | Protocols                            | Firmwares                | Notes                                                 |
+| ---------------------------- | ------------------------------------ | ------------------------ | ----------------------------------------------------- |
+| [ARK 4IN1 ESC]               | [Dshot], [PWM]                       | [AM32]                   | Has versions with/without connnectors                 |
+| [Holybro Kotleta 20]         | [DroneCAN], [PWM]                    | [PX4 Sapog ESC Firmware] |                                                       |
+| [Vertiq Motor & ESC modules] | [Dshot], [OneShot], Multishot, [PWM] | Vertiq firmware          | Larger modules support DroneCAN, ESC and Motor in one |
+| [VESC ESCs]                  | [DroneCAN], [PWM]                    | VESC project firmware    |                                                       |
+| [Zubax Telega]               | [DroneCAN], [PWM]                    | Telega-based             | ESC and Motor in one                                  |
 
-### Oneshot 125
+<!-- Links for table above -->
 
-[OneShot 125 ESCs](../peripherals/oneshot.md) are usually much faster than PWM ESCs, and hence more responsive and easier to tune.
-They are preferred over PWM for multicopters (but not as much as [DShot ESCs](#dshot), which do not require calibration, and may provide telemetry feedback).
-There are a number of variants of the OneShot protocol, which support different rates.
-PX4 only supports OneShot 125.
+[ARK 4IN1 ESC]: ../esc/ark_4in1_esc.md
```

Comment:
FYI this is reference link format. I only use it rarely, but it can be useful when you want to make a table easier to edit, because it shrinks the width. Note table is now alphabetic order, and it always puts PWM last - because even though it is the most common ESC format, we want to recommend Dshot or maybe DroneCAN by preference.